vertx-mysql-client/src/main/java/io/vertx/mysqlclient/impl/codec/ExtendedBatchQueryCommandCodec.java

```diff
@@ -47,6 +47,11 @@ private void doExecuteBatch() {
     if (batchIdx < params.size()) {
       sequenceId = 0;
       Tuple param = params.get(batchIdx);
+      // binding parameters
+      String bindMsg = statement.bindParameters(param);
+      if (bindMsg != null) {
+        completionHandler.handle(CommandResponse.failure(bindMsg));
```

Comment:
don't we need a return statement after this failure ?

Reply:
fixed in https://github.com/eclipse-vertx/vertx-sql-client/pull/720/commits/612738b570cb98434d2bc223c793f2dfc81b553b